rpcs3/Emu/Cell/Modules/cellSearch.cpp

```diff
@@ -311,6 +364,10 @@ error_code cellSearchStartListSearch(CellSearchListSearchType type, CellSearchSo
 						info.duration = 0;
 						strcpy_trunc(info.title, item.name);
 						strcpy_trunc(info.artistName, "ARTIST NAME");
+						// cellSearch.notice("cellSearchStartListSearch(): info.title: \"%s\"", info.title);
```

Comment:
either add it or not. but these comments are useless to others.
if you just added it for debugging, then it's better to not commit it at all.